MovieApp/PresentationLayer/Favorites/FavoritesViewController.swift

```diff
@@ -8,12 +8,16 @@ class FavoritesViewController: UIViewController {
 
     private let inset: CGFloat = 20
     private let verticalSpacing: CGFloat = 35
+
     private let presenter: FavoritesPresenter
 
     private var movies: [DetailedMovieViewModel] = []
 
-    init(presenter: FavoritesPresenter) {
+    private weak var router: MovieDetailsRouterProtocol?
```

Comment:
Router does not need to be declared as weak var. It is a singleton and should exist as the app exists.